### PR TITLE
Fix task model updated_date field: auto_now = True

### PR DESCRIPTION
**Pull request by JK-97**
### Motivation and context

&nbsp;&nbsp;After creating a task, add a label attribute, and then export the data set. The exported annotations does not contain the new attribute. 

&nbsp;&nbsp;The reason is that when the attributes are changed, the update time of the task has not changed.
and export the task dataset cache

&nbsp;&nbsp;Normally, after changing the attributes, the exported annotation information should contain the added or changed attributes

&nbsp;&nbsp;Related code:
cvat/apps/dataset_manager/views.py:line 56.  func export_task()
```
        if not (osp.exists(output_path) and \
                task_time <= osp.getmtime(output_path)):`

```
cvat/apps/engine/serializers.py:line 323.   func update()
```
class TaskSerializer
    ....
    def update(self, instance, validated_data):
        ...
        instance.save()
        return instance
```

### How has this been tested?

yes,
### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: cvat/apps/engine/models.py [hunk 1]
```diff
@@ -167,7 +167,7 @@ class Task(models.Model):
         on_delete=models.SET_NULL, related_name="assignees")
     bug_tracker = models.CharField(max_length=2000, blank=True, default="")
     created_date = models.DateTimeField(auto_now_add=True)
-    updated_date = models.DateTimeField(auto_now_add=True)
+    updated_date = models.DateTimeField(auto_now=True)
```
Comment (azhavoro) [inline cvat/apps/engine/models.py:170]: This change requires creating a migration. Could you please add a migration file?

**Review by azhavoro — CHANGES_REQUESTED** [review]

**Comment by azhavoro** [comment]
@JK-97 
Hi, thanks for the contribution!

**Comment by nmanovic** [comment]
@JK-97 , thanks for the pull requests. Could you please update it in accordance with comments? Need to add migration file. You can generate it using `python3 manage.py makemigrations` command.

**Comment by bsekachev** [comment]
Good day, @JK-97 

What are your plans on this PR?

File: cvat/apps/engine/models.py (hunk 1 above)
Comment (JK-97) [reply cvat/apps/engine/models.py:170]: resolved! pls review again!

**Review by azhavoro — APPROVED** [review]

**Comment by bsekachev** [comment]
Thank you so much for the contribution!

**Pull request merged**